llm_bench/python/llm_bench_utils/bench_classification.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
do we really need that? We do not use llm bench for classification models benchmarking, maybe it is good time for cleanup?

Reply:
cleanup.